config.js

```diff
@@ -90,14 +90,14 @@ module.exports = {
     // projects.
     ignoreContextHeader: false,
 
+    // The contents of a key file. If this field is set, its contents will be
+    // used for authentication instead of your application default credentials.
+    credentials: null,
+
```

Comment:
Why is this change here? It don't seem to to have anything to do with the rest of the changes in this PR.